### ISSUE TITLE
False positive result when __DIR__ is used 

### ISSUE THREAD

**Issue by mimmi20** [Roave/BackwardCompatibilityCheck#400]
Example:

```php
final class EngineLoaderFactory implements EngineLoaderFactoryInterface
{
    public const DATA_PATH = __DIR__ . '/../../data/engines';
}
```

Expected Result: no error

Result:
```shell
# Changed
 - [BC] Value of constant BrowserDetector\Loader\EngineLoaderFactory::DATA_PATH changed from '/tmp/api-compare-ed688120622a186b52c6e1b33a3065507a5f04e961daa0714f8ee/src/Loader/../../data/engines' to '/tmp/api-compare-b5c256f5ff06bbf49461563348993ebb614d660961daa073ba0ab/src/Loader/../../data/engines'
```

**Comment by asgrim** [comment]
Hmm. Constant values changing is only a BC break if the values are used anywhere as plain strings. I think perhaps a way to ignore some BC breaks might be the way forward here, WDYT @Ocramius 

Maybe we could add an attribute, something like:

```php
#[\Roave\BackwardCompatibilityCheck\Attribute\IgnoreBreak(ConstantValueChanged::class)]
public const THE_CONST_HERE = __DIR__ . '/blah';
```

**Comment by Ocramius** [comment]
Tricky one: I guess we need to handle `__DIR__` differently, but we don't have the abstraction to do so in this library 🤔
